2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -656,7 +656,67 @@ Gli obiettivi dell'ottavo sprint riguardano tutti gli aspetti del progetto, di c
 === Diagramma di Gantt
 [da inserire]
 
+== Undicesimo Sprint dal 14-01-2024 al 21-01-2024
 
+=== Obiettivi prefissati
+
+Gli obiettivi dell'undicesimo sprint si incentrano sulla preparazione per il colloquio RTB oltre che sul ricontrollo di alcuni aspetti di attività già concluse. In particolare gli obiettivi prefissati per questo sprint sono:
+
+- aggiornare il Piano di Qualifica con nuove metriche individuate riguardanti:
+  - budget utilizzato;
+  - ore rimanenti.
+- ultimare il documento Analisi dei Requisiti con alcune modifiche minori agli use cases;
+- verificare la correttezza di quanto prodotto nei documenti redatti finora;
+- preparare la candidatura per il colloquio RTB.
```

Comment:
Questo punto può essere sviluppato in più punti:
- Redigere nuova lettera di presentazione con aggiornamento di preventivo
- Preparare presentazione
- Continuare la revisione dei documenti